cfgov/unprocessed/css/skip-nav.less

```diff
@@ -20,8 +20,13 @@
 }
 
 #skip-nav:focus {
```

Comment:
Would it be better to add the `a-btn a-btn__super` classes to `#skip-nav` than to have them duplicated in the output here?